src/app/faq-page/faq-page.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="faq-page standard-content-container-width">
+  <app-header></app-header>
```

Comment:
See comment on app.routing.module.ts on how to omit <app-header>, <app-footer> and such.

Reply:
This also leads to the header and footer not taking up the entire width

![image](https://user-images.githubusercontent.com/38329766/141661268-2232be5b-1888-4fd8-83c2-580277391ef2.png)

Reply:
Done